PureBasicIDE/ColorSchemes.pb

```diff
@@ -0,0 +1,324 @@
+﻿; --------------------------------------------------------------------------------------------
+;  Copyright (c) Fantaisie Software. All rights reserved.
+;  Dual licensed under the GPL and Fantaisie Software licenses.
+;  See LICENSE and LICENSE-FANTAISIE in the project root for license information.
+; --------------------------------------------------------------------------------------------
+
+
+; Store color names in an array, for indexable lookup
+Global Dim ColorName.s(#COLOR_Last_IncludingToolsPanel)
+
+; Color Scheme structure
+Structure ColorSchemeStruct
+  Name.s
+  File.s
+  
+  ColorValue.l[#COLOR_Last_IncludingToolsPanel + 1]
+  
+  IsIDEDefault.i
+  IsAccessibility.i
+EndStructure
+
+; Global list of built-in / found schemes
+Global NewList ColorScheme.ColorSchemeStruct()
+
+; Color values with special meanings
+#ColorSchemeValue_UseSysColor = -1
+#ColorSchemeValue_Undefined   = -2
+
+
+; These (from Preferences.pb) are used
+Global PreferenceToolsPanelFrontColor, PreferenceToolsPanelBackColor
+Declare UpdatePreferenceSyntaxColor(ColorIndex, Color)
+Declare UpdateImageColorGadget(Gadget, Image, Color)
+
+
+; Returns #True if the specified color scheme matches the user's current color settings, otherwise #False
+Procedure.i ColorSchemeMatchesCurrentSettings(*ColorScheme.ColorSchemeStruct)
```

Comment:
Integer is the default type. So adding it to procedure declarations is unneeded. The same goes for variable declarations.

The general style in the IDE sources is:
* Integer variables without any type declaration
* String variables always in the `Variable$` style
* Float or double variables always with type (even outside of the declaration)